homeassistant/components/matter/lock.py

```diff
@@ -99,7 +99,7 @@ async def async_lock(self, **kwargs: Any) -> None:
             # the lock should acknowledge the command with an attribute update
             # but bad things may happen, so guard against it with a timer.
             self._optimistic_timer = self.hass.loop.call_later(
-                5, self._reset_optimistic_state
+                30, self._reset_optimistic_state
```

Comment:
We never test that the callback runs after the time.

To make it easier to test this make the time a module level constant that can easily be patched.